### PR TITLE
Posix compat

### PR DESCRIPTION
**Pull request by ShellyHerself**
Likely not complete yet. Mainly for Mozzarilla Purposes.

The filepicker library needs to be tested yet, which is separate from this.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -13,3 +13,9 @@
 from binilla import constants
 
 constants.inject()
+
+# Hack: Temp import filedialog before anything else to avoid import order
+# conflict to do with FieldWidget.
+# TODO: Moses, please fix the root of this issue.
+from binilla.windows import filedialog
+del filedialog
```
Comment (ShellyHerself) [inline __init__.py]: Any way we can fix this @MosesofEgypt?

File: defs/config_def.py [hunk 2]
```diff
@@ -106,7 +107,10 @@
     {NAME: "scroll_unselected", TOOLTIP: ttip.field_widget_scroll_unselected}, # RENAMED
     {NAME: "evaluate_entry_fields", TOOLTIP: ttip.field_widget_evaluate_entry_fields},
     {NAME: "show_structure_meta", TOOLTIP: ttip.field_widget_show_structure_meta},
-    DEFAULT=sum([1<<i for i in (2, 3, 4, 6, 7, 8, 9, 10)])
+    DEFAULT=(
+        sum([1<<i for i in (2, 3, 4, 6, 7, 8)])
+        if e_c.IS_LNX else
+        sum([1<<i for i in (2, 3, 4, 6, 7, 8, 9, 10)]))
```
Comment (ShellyHerself) [inline defs/config_def.py]: A comment explaining this would be nice.

File: widgets/field_widgets/color_picker_frame.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import threadsafe_tkinter as tk
 import tkinter.ttk as ttk
 
-from tkinter.colorchooser import askcolor
+from tkcolorpicker import askcolor
```
Comment (ShellyHerself) [inline widgets/field_widgets/color_picker_frame.py:4]: Add `tkcolorpicker` to the dependencies.

**Review by ShellyHerself — COMMENTED** [review]
Some personal notes.

File: __init__.py (hunk 1 above)
Comment (MosesofEgypt) [reply __init__.py]: Will try to look into this today/tonight
EDIT: Fixed. Will push changes tonight.

File: defs/config_def.py (hunk 2 above)
Comment (MosesofEgypt) [reply defs/config_def.py]: Done. Will be included in update tonight.

**Pull request merged**